boilerplate/scripts/build.js

```diff
@@ -1,8 +1,8 @@
 import builder from "statinamic/lib/builder"
 
-import * as layouts from "../web_modules/layouts"
-import metadata from "../web_modules/app/metadata"
-import routes from "../web_modules/app/routes"
+const layouts = require.resolve("../web_modules/layouts")
+const metadata = require.resolve("../web_modules/app/metadata")
+const routes = require.resolve("../web_modules/app/routes")
 import store from "../web_modules/app/store"
```

Comment:
Nope. The store must be in memory and hold data. If we include it, this will become a big refactor